src/pages/Products/List.js

```diff
@@ -0,0 +1,20 @@
+/*eslint-disable*/
+import { useEffect } from 'react';
+import { useState } from 'react';
+import { Link } from 'react-router-dom';
+import dataList from './DataList';
+import ProductImg from './ProductImg/ProductImg';
```

Comment:
```suggestion
import { useEffect, useState } from 'react';
import { Link } from 'react-router-dom';
import ProductImg from './ProductImg/ProductImg';
import dataList from './DataList';
```

import 순서 수정해주세요! 일반적인 convention을 따라 순서만 잘 지켜주셔도 가독성이 좋아집니다. 
아래 순서 참고해주세요.

React → Library(Package) → Component → 변수 / 이미지 → css 파일(scss 파일)